integrations/mobly/chip_device.py

```diff
@@ -0,0 +1,61 @@
+import os
+from pathlib import Path
+import serial
+
+from pw_hdlc_lite.rpc import HdlcRpcClient
+
+# Point the script to the .proto file with our RPC services.
+PROTO = Path(os.environ['PW_ROOT'], 'pw_rpc/pw_rpc_protos/echo.proto')
+
+MOBLY_CONTROLLER_CONFIG_NAME = 'ChipDevice'
+
+class Error(Exception):
```

Comment:
please consider auto stylize the python code, I am using https://black.readthedocs.io/en/stable/the_black_code_style.html

Reply:
done